src/app/shared/notification/service/notification.service.ts

```diff
@@ -45,4 +51,12 @@ export class NotificationService {
   public onFeatureNotAvailableYet() {
     this.addHintNotification('This feature is not available yet. But it is coming soon! :-)');
   }
+
+  public rateComplexNavigator(): void {
+    this.addRatingNotification('<div><input class="input-small" value="textbox"/><a href="' + environment.complex_portal_support_url + '">Tell us about your experience with the Complex Navigator!</a></div>');
+  }
+
+  public complexNavigatorAnnouncement() {
```

Comment:
At the moment the pop up displays at the very top, and it's easy to miss. We can configure it to display lower so it displays on the white part of the screen.

1. In `styles.css` add the following class:
```
.toast-right-under-header {
  right: 10px;
  top: 225px;
}
```

2. Update this method to use that position class:
```
  public complexNavigatorAnnouncement() {
    this.addAnnouncementNotification(
      'Try out our new display: the Complex Navigator!',
      {
        positionClass: 'toast-right-under-header'
      });
  }
```

Give it a try, and you can also use it for the `rateComplexNavigator` if you want.

Reply:
There are more options, like `disableTimeOut` and `closeButton`. So we can disable the timeout and force users to have to close the pop up.